lib/rubocop/cop/mixin/queries_helper.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    # Common functionality related to Active Record queries checks.
+    module QueriesHelper
+      WHERE_METHODS = %i[where rewhere].freeze
+
+      def in_where?(node)
+        send_node = node.each_ancestor(:send).first
+        send_node && WHERE_METHODS.include?(send_node.method_name)
+      end
```

Comment:
Maybe this method (and `WHERE_METHODS ` constant) can be added to `RuboCop::Cop::ActiveRecordHelper`. What do you think about it?
https://github.com/rubocop-hq/rubocop-rails/blob/v2.6.0/lib/rubocop/cop/mixin/active_record_helper.rb